### DISCUSSION TITLE
Si4432 GPIO pins

### DISCUSSION THREAD

**Discussion by radio-satellites** [jgromes/RadioLib#1223] [Q&A] [answered]
Hello, 

To transmit AFSK with the Si4432, which of the GPIOs (0,1,2) do I need to connect to the microcontroller? The examples only list "GPIO." 

--Sasha

**Comment by jgromes** [comment, marked as answer]
Hmm, good question - it's been a while since I last used that module. Looking into the source, `Si443x::directMode` sets GPIO1 as clock output and GPIO2 as data input, so I think GPIO2.

**Reply by radio-satellites** [reply]
Thanks! :)

**Reply by radio-satellites** [reply]
One more question @jgromes: the Si4432 has programmable loading capacitors. Does radiolib support programming these? If not, what is the default that it's left at? How should I go about connecting a crystal to it?

**Reply by jgromes** [reply]
This feature is not implemented in RadioLib, so you should consult the datasheet on that.

However, even though this is not implemented in RadioLib at the moment, you can use [God mode](https://github.com/jgromes/RadioLib/wiki/God-Mode) (or the slightly less intrusive Low-level access mode) to gain direct access to the SPI registers from your user program.
